python/taichi/examples/rendering/oit_renderer.py

```diff
@@ -70,12 +70,12 @@ def intersect_sphere(light: ti.template(), sphere: ti.template()):
             if t1 > 0:
                 hit_pos1 = light.pos + light.dir * t1
                 dist1 = t1
-                normal1 = (hit_pos1 - sphere.center).normalized()
+                normal1 = normalize((hit_pos1 - sphere.center)
```

Comment:
```suggestion
                normal1 = normalize(hit_pos1 - sphere.center)
```
I'm sorry this '(' was not closed...

Reply:
fixed